eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/http/async/CompleteHandler.java

```diff
@@ -22,6 +22,7 @@
  *
  * @param <T>
  */
+@FunctionalInterface
 public interface CompleteHandler<T> {
```

Comment:
It seems that there's no lambda usage of this class. What does the addition of this annotation bring?